src/base/ReactorStructs.sol

```diff
@@ -49,6 +70,16 @@ struct ResolvedOrder {
     bytes32 hash;
 }
 
+/// @dev generic concrete order that specifies exact tokens which need to be sent and received
+struct ResolvedOrderV2 {
+    OrderInfoV2 info;
+    InputToken input;
```

Comment:
For more flexibility, should we allow an array of input tokens? Could be used to batch liquidate all tokens to  ETH for example.